aws/elasticache_redis/main.tf

```diff
@@ -0,0 +1,80 @@
+terraform {
```

Comment:
Regarding `aws/elasticache_redis/` directory, for now I put it in the same directory as `aws`. But that might make the root repo too long with lot of directories. So what I'll do it after merging this and another PR (the ECS+RDS stitcher) I'll refactor and move them into something like `examples/ecs_rds_stitcher` similar to how we did with CDK. But for now I wanted to focus this PR on just ElastiCache Redis.

Reply:
sounds like a good plan